cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/centroid/help.xhtml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Centroid</title>
+</head>
+<body>
+<div id ="help">
+	<h4>General explanation</h4>
+	<p>
+This function is used to create a centroid (or geometric center) for a given source dataset and to map it to the target dataset.<br> 
+</br>
+<br> 
+</br>
+In the example below, the centroid function was applied to generate the centroid for the protected sites target dataset. Note that for mapping 
```

Comment:
Protected Sites maybe in upper case because it that is how it is usually handled in INSPIRE?